AEPCore/Sources/core/MobileCore+Tracking.swift

```diff
@@ -0,0 +1,51 @@
+/*
+ Copyright 2020 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+/// Defines the public interface for the Tracking methods
+@objc
+public extension MobileCore {
+    /// Generates and dispatches a track action `Event`
+    /// - Parameters:
+    ///   - action: `String` representing the name of the action to be tracked
+    ///   - data: Dictionary of data to attach to the dispatched `Event`
+    @objc(trackAction:data:)
+    static func trackAction(_ action: String?, data: [String: Any]?) {
```

Comment:
Any reason the first parameter names are omitted when calling the API? `_ action: String?` vs `action: String?`

Reply:
`action` is the last word in the method name, and also the parameter name.  i think in this case the label is implied, so that's why i omitted.  thoughts?

Reply:
got it, makes sense to me 👍

Reply:
static func track(action: String?, data: [String: Any]?) 
static func track(state: String?, data: [String: Any]?) 

How about this?